services/src/handlers/ebv.rs

```diff
@@ -15,16 +15,24 @@ use log::debug;
 use serde::Serialize;
 use snafu::{ResultExt, Snafu};
 use std::path::PathBuf;
-
-pub(crate) fn init_ebv_routes<C>(base_url: Option<String>) -> Box<dyn FnOnce(&mut ServiceConfig)>
+use url::Url;
+
+/// Initialize ebv routes
+///
+/// # Panics
+/// This route initializer panics if `base_url` is `None` and the `ebv` config is not defined.
+///
+pub(crate) fn init_ebv_routes<C>(base_url: Option<Url>) -> Box<dyn FnOnce(&mut ServiceConfig)>
 where
     C: Context,
     C::Session: FromRequest,
 {
     Box::new(move |cfg: &mut web::ServiceConfig| {
-        cfg.app_data(web::Data::new(BaseUrl(
-            base_url.unwrap_or_else(|| BASE_URL.to_owned()),
-        )))
+        cfg.app_data(web::Data::new(BaseUrl(base_url.unwrap_or_else(|| {
```

Comment:
why not get the base_url from config inside requests like we always do? is it s.t. it panics on startup?

Reply:
The option was introduced due to testing. Currently it just panics on startup instead of throwing an error at runtime.
I currently see no benefit in calling the config thing over and over again, but if there is something that I don't have on my mind, I can change it.